client/js/store/modules/game.js

```diff
@@ -71,6 +72,33 @@ const initialState = resetState();
 export default {
   state: initialState,
   getters: {
+    // TODO: Replace this with an actual deck array :)
+    cards(state, getters) {
```

Comment:
Let's further discuss the intended consumption of this getter. My instinct is that a dictionary/map would be more appropriate than an array to look up by id for example in constant time, but it depends on the usage. 

That said, I'm still of the opinion that the full list of cards may be unnecessary for the selection process, because the data you need in order to display a card and to determine what moves it can do is the cards suit and rank, both of which are already available on the individual card component that gets clicked. So it's not fully clear to me what we'll do with this full list. Let's discuss